app/models/setting/general.rb

```diff
@@ -23,7 +23,8 @@ def self.load_defaults
         self.set('db_pending_migration', N_("Should the `foreman-rake db:migrate` be executed on the next run of the installer modules?"), true, N_('DB pending migration')),
         self.set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true, N_('DB pending seed')),
         self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60, N_('Proxy request timeout')),
-        self.set('login_text', N_("Text to be shown in the login-page footer"), nil, N_('Login page footer text'))
+        self.set('login_text', N_("Text to be shown in the login-page footer"), nil, N_('Login page footer text')),
+        self.set('host_power_status', N_("Show power status on host index page. Enabling this feature may lead to decreased performance of the page."), false, N_('Show host power status'))
```

Comment:
does it really? I mean its an ajax call, the page will render initially as it is today?